README.md

```diff
@@ -59,6 +59,20 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
   * To edit, make edits in the editor and save it.  When prompted to upload the file, click 'Upload'.
   * CAUTION: Manually editing your Web App's files could cause unexpected behavior.
 
+## Managing Azure Subscriptions
+
+If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
+
+![Sign in to Azure](resources/SignInScreenshot.png)
+
+If you don't have an Azure Account, you can sign up for one today for free and receive $200 in credits by selecting "View->Command Palette" and searching for "Azure: Create an Account".
+
+You may sign out of Azure by selecting "View->Command Palette" and searching for "Azure: Sign Out".
+
+To select which subscriptions show up in the extension's explorer, click on the "Select Subscriptions..." button on any subscription node (indicated by a "key" icon when you hover over it), or select "View->Command Palette" and search for "Azure: Select Subscriptions". Note that this selection affects all VS Code extensions that support the [Azure Account and Sign-In](https://github.com/Microsoft/vscode-azure-account) extension.
```

Comment:
Its not indicated by a "key" icon - its indicated by a "filter" icon. You'll have to update the screenshot, too